appveyor.yml

```diff
@@ -2,19 +2,11 @@ environment:
   RUST_TEST_THREADS: 1
   matrix:
   - TARGET: x86_64-pc-windows-msvc
-    RUST_CHANNEL: 1.15.0
+    RUST_CHANNEL: stable
```

Comment:
Unfortunately you need to use a specific version number here - the script pulls from `https://static.rust-lang.org/dist/rust-${env:RUST_CHANNEL}-${env:TARGET}.exe`, and there's no stable file in there.

Reply:
Oh, I am sorry. I'll fix it.